app/models/purl.rb

```diff
@@ -20,6 +20,29 @@ class Purl < ActiveRecord::Base
   }
 
   ##
+  # Return true targets with always values only if the object is not deleted in
+  # purl mount
+  # @return [Array]
+  def computed_true_targets
+    return [] unless deleted_at.nil?
+    true_targets | Settings.ALWAYS_SEND_TRUE_TARGET.to_a
```

Comment:
Is that a bitwise or operator?  I've never used one and am curious - if there a reason to use that instead of the logical || operator?